### PR TITLE
first cut errata

### PR DESCRIPTION
**Pull request by kkraune**
FYI @geirst

### CODE REVIEW

File: ERRATA.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!-- Copyright verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+## Errata
+
+### 2020-11-30: Document inconsistency
+This bug existed between 2020-08-28 and 2020-09-23.
```
Comment (geirst) [inline ERRATA.md:6]: We should probably use versions here. Introduced in Vespa 7.277.38. Fixed in 7.292.82.

File: ERRATA.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  at least one attribute vector is not flushed to the current serial number.
+* Due to the bug, replay of the transaction log will fail to replay feed operations to attributes after replaying the config change.
+  The effect is that all attributes that were not flushed as part of prepare for restart
+  will not get any of the updates since the last time they were flushed.
+* If a document was previously removed, lid space compaction will move another document to that local document id.
+  Due to later missing updates as part of restarting we might see values from the removed document for some of the attributes.
+* When the problem attributes are later flushed this inconsistency will be permanent.
+
+Solution:
+* Upgrade Vespa to at least vespa-7.306.19-1.el7.x86_64.rpm.
```
Comment (geirst) [inline ERRATA.md:22]: This was fixed in 7.292.82.

**Review by vekterli — APPROVED** [review]

File: ERRATA.md (hunk 1 above)
Comment (kkraune) [reply ERRATA.md:6]: better!

File: ERRATA.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<!-- Copyright verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
+
+## Errata
+
+### 2020-11-30: Document inconsistency
+This bug existed between 2020-08-28 and 2020-09-23.
+The following needs to happen to trigger the bug:
+
+* Visibility delay is non-zero.
```
[the remark this answers is not in the document]
Comment (kkraune) [reply ERRATA.md:9]: yes, good clarification, will add

**Pull request merged**